app/views/user_mailer/analytics_report.html.erb

```diff
@@ -0,0 +1,193 @@
+<html>
+
+<head>
+  <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  <style>
+    main {
+      max-width: 676px;
+      margin: auto;
+      font-family: Arial, Helvetica, sans-serif;
+    }
+
+    table {
+      width: 100%;
+      margin-bottom: 50px;
+      border-collapse: collapse;
+    }
+
+    th {
+      text-align: left;
+    }
+
+    .styled-table thead tr {
+      background-color: #212529;
+      color: #ffffff;
+      text-align: left;
+    }
+
+    .styled-table th,
+    .styled-table td {
+      padding: 12px 15px;
+      border-bottom: 1px solid rgb(222, 226, 230);
+    }
+
+    .users {
+      overflow: auto;
+    }
+  </style>
+</head>
+
+<body>
+  <main>
+    <h1>Periodic reports for visitor analytics</h1>
+    <div>
+      <h2>
+        Call to Actions
+      </h2>
+
+      <table class="styled-table">
+        <thead>
+          <tr>
+            <th>CTA</th>
+            <th>Response count</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% response_total=0 %>
+            <% @report[:ctas] do |cta|%>
+              <tr>
+                <td><a href=<%="/call_to_actions/#{cta[:id]}" %>"><%= cta[:title] %></a></td>
+                <td>
+                  <%= cta[:response_count] %>
+                    <% response_total=+ cta[:response_count] %>
+                </td>
+              </tr>
+              <%end%>
+        </tbody>
+
+        <tfoot>
+          <tr>
+            <td><strong>Total</strong></td>
+            <td>
+              <%= response_total %>
+            </td>
+          </tr>
+        </tfoot>
+      </table>
+    </div>
+
+    <% @report[:visits].each do |visited_by, response|%>
+      <div>
+        <h2>
+          Visits by <%= visited_by.to_s.humanize%>
+        </h2>
+
+        <table class="styled-table">
+          <thead>
+            <tr>
+              <th><%= visited_by.to_s.humanize%></th>
+              <th>Response count</th>
```

Comment:
this text should not be hard coded. It should read something like: 

Visits by Country/region/city/domain etc => visits
Call to Actions => responses